machine_learning/DegradationModelBlackbox.py

```diff
@@ -512,113 +547,249 @@ def __init__(
         self.width = width
         self.n_channels = n_channels
 
+    def call(self, params, training = False):
+        """
+        Call function for the Model during training or evaluation.
+
+        Examples:
+
+            training:
```

Comment:
these calls have tuple arguments.
but down in test_single_voltages, the call is made with a dictionary...